src/host.cpp

```diff
@@ -1011,7 +1011,11 @@ namespace rhost {
                         continue;
                     }
 
+#ifdef _WIN32
                     strcpy_s(buf, len, s.c_str());
+#else
+                    strcpy(buf, s.c_str());
```

Comment:
I would suggest that we make our own implementation of `strcpy_s` here. For now it can be just a simple wrapper over `strcpy` that ignores the buffer length parameter, but later we would add proper checking to it. Ditto for other `_s` functions.